firestore/firestore_quickstart/main.go

```diff
@@ -15,7 +15,7 @@
 // Sample firestore_quickstart demonstrates how to connect to Firestore, and add and list documents.
 package main
 
-// [START fs_initialize]
+// [START firestore_setup_client_create]
```

Comment:
Thanks for opening this PR @AlisskaPie, we actually need to keep the existing tags and add the new tags alongside right now. At a later time we'll be able to come back and cleanup the existing tags.

Can you update the PR to look like the following:

```suggestion
// [START fs_initialize]
// [START firestore_setup_client_create]
```

Reply:
Thank you for quick review, done!
Also, there're 5 samples that were not implemented on a site yet. I did them recently, its query watch samples (
fs_detach_listener, 
fs_listen_document,
fs_listen_errors,
fs_listen_changes,
fs_listen_multiple)
I guess we can safely rename these tags.

Reply:
I think this would be okay. @crwilcox Do you agree?

Reply:
I agree. Are these a match for a rows in the tracking spreadsheet? If not can we add rows? We are going to want to convert all `fs_name` snippets in the end.

Reply:
Yes, they are the same.
The one region tag left not replaced is `fs_update_multiple_fields` tag. There was no new region tag proposed for it in the tracking spreadsheet.